### ISSUE TITLE
removing all proficiencies throws TypeError

### ISSUE THREAD

**Issue by phivk** [lowmess/metalsmith-resume#16]
When I remove all proficiencies (as suggested in `person.yaml`), like such
```
proficiencies:
  # - list item
```
the build process throws a throws a TypeError, which prevents the resume being built:
```
{ TypeError: metalsmith-resume/templates/default.pug:32
    30|               +experience(collections.education[i])
    31| 
  > 32|         if person.proficiencies.length > 0
    33|           +section
    34|             +left-column
    35|               +section-title('Proficiencies')

Cannot read property 'length' of null
    at eval (eval at wrap (metalsmith-resume/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:526:25)
````

I'm on OSX 10.12.6 using latest node (5.5.1)

**Comment by phivk** [comment]
I found a simple fix in `default.pug` and submitted this PR:
https://github.com/lowmess/metalsmith-resume/pull/17

**Comment by lowmess** [comment]
Fixed with #17
